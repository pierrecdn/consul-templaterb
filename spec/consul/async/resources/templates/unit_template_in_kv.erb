This example fetch kv that contains template with = 5 * 2
It should display 10
value: <%= render_from_string(kv('template-in-template/return-10').get_value_decoded) %>