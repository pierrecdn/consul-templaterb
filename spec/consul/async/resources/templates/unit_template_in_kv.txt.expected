This example fetch kv that contains template with = 5 * 2
It should display 10
value: 10
